views/homepage.handlebars

```diff
@@ -1,9 +1,72 @@
-{{#each Search}}
-<div class="mx-auto justify-center movie sm:h-400 lg:h-530 w-375 text-center">
-    <img class="mx-auto sm:h-1/2 md:h-5/6 lg:h-5/6 w-5/6"
-        src= {{Poster}}>
-    <button class="mx-auto my-auto to-watch">To Watch</button>
-    <button class="mx-auto my-auto watched">Watched</button>
-</div>
```

Comment:
A little confused because it looks like Handlebars is being removed and not added back in, just want to make sure this was what you intended

Reply:
I had to move handlebars around a little.  I added a search.handlebars and changed homepage.handlebars.  I verified both pages worked before doing a pull request.